index.d.ts

```diff
@@ -786,7 +786,12 @@ declare namespace plugins {
    * This plugin automatically instruments the
    * [cucumber](https://www.npmjs.com/package/@cucumber/cucumber) module.
    */
-  interface cucumber extends Instrumentation {}
+  interface cucumber extends Instrumentation {
```

Comment:
Why is only `cucumber` analyzable (as defined by `Instrumentation`).

Reply:
probably an oversight. I can't think of anything else